engine/src/main/scala/cromwell/webservice/MetadataBuilderActor.scala

```diff
@@ -235,25 +235,36 @@ object MetadataBuilderActor {
 
 }
 
-class MetadataBuilderActor(serviceRegistryActor: ActorRef) extends LoggingFSM[MetadataBuilderActorState, Unit] with DefaultJsonProtocol {
+private[webservice] trait MetadataBuilderActorData
+private[webservice] case object NoData extends MetadataBuilderActorData
+private[webservice] case class ActionBeingAwaited(action: MetadataServiceAction) extends MetadataBuilderActorData
```

Comment:
it's so that the `MetadataBuilder` can remember whether it made a single- or multiple- metadata request to the `MetadataService`